sources/views/contacts.js

```diff
@@ -1,42 +1,99 @@
 import { JetView } from "webix-jet";
 import { contacts } from "../models/contacts";
+import FormView from "./form";
 
 export default class Contacts extends JetView {
 	config() {
-
+		const _ = this.app.getService("locale")._;
 		return {
 			type: "wide",
+			localId: "contacts",
 			cols: [
 				{
-					view: "list",
-					localId: "contactsList",
-					scrollX: false,
-					select: true,
-					template: "#Name# - #Email#",
-					css: "webix_shadow_medium app_start"
-				},
-				{
-					view: "form",
-					margin: 20,
-					elements: [
-						{ view: "text", label: "Name" },
-						{ view: "text", type: "email", label: "Email" },
-						{ view: "text", type: "text", label: "Status" },
-						{ view: "text", type: "text", label: "Country" },
+					type: "form",
+					rows: [
+						{
+							view: "list",
+							localId: "contactsList",
+							scrollX: false,
+							select: true,
+							template: function (obj) {
+
+								return (
+									"<div class='space'> <div>" + obj.Name + "  " + obj.Email + "</div> " +
+									"<span class='removeBtn webix_icon wxi-trash'></span></div>"
+								);
+							},
+							css: "webix_shadow_medium app_start",
+							click: function (id) {
+								this.$scope.setParam("id", id, true);
+							},
+							onClick: {
+								removeBtn: function (ev, id) {
+									contacts.remove(id);
+
+									const contactsList = this.$scope;
+									let firstId = this.getFirstId();
+
+									if (firstId) {
+										this.select(firstId);
+										contactsList.setParam("id", firstId, true);
```

Comment:
no, you do not need to do this manually every time
the list controls the form via the url param
list.select(someId) is called -> add an event listener for list: this.on(list, "onAfterSelect" => this.setParam(...))
the same for unselect, but there you need to reset the param
so maybe a common onSelectChange event is better